platform/ui/src/components/AllInOneMenu/Item.tsx

```diff
@@ -20,6 +22,8 @@ const Item = ({ label, secondaryLabel, icon, onClick }: ItemProps) => {
     <div
       className="all-in-one-menu-item all-in-one-menu-item-effects"
       onClick={onClickHandler}
+      onMouseEnter={onMouseEnter}
```

Comment:
is this for hover previews? add a comment please

Reply:
Yes